content/data-types/studio/_index.md

```diff
@@ -0,0 +1,78 @@
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Get cozy with Java syntax by writing a console program that calculates the area of a circle based on input from the user.
+
+## Creating your class
+Since you’re still new to Java and IntelliJ, we’ll provide some extra direction the first studio.
+
+First, make a new folder, or package, to hold your studio work. Create a new package named `org.launchcode.java.studios.areaofacircle` by right-clicking (or ctrl-clicking for some Mac users) on the `src` directory in `java-web-dev-exercises` and selecting _New_ > _Package_. Be sure to enter the full name, or your package won’t be created in the correct location.
+
+Create your class in the `java-web-dev-exercises` project within the package `org.launchcode.java.studios.areaofacircle` by right-clicking/ctrl-clicking on the `studios.areaofacircle` package/folder and selecting _New_ > _Java Class_. Enter the name Area. Select the option to add the file to Git when the window appears.
+
+## A. The `Area` Class
+
+Write a class, `Area`, that prompts the user for the radius of a circle and then calculate its area and print the result.
+
+{{% notice green "Tip" "rocket" %}} 
+ Recall that the area of a circle is A = pi * r * r where pi is 3.14 and r is the radius.
+
+ Unlike some other languages, Java does not have an exponent operator.
+{{% /notice %}}
+
+Here’s an example of how your program should work:
+```bash
+Enter a radius: 
+2.5
+The area of a circle of radius 2.5 is: 19.625
+```
+
+Some questions to ask yourself:
+
+1. What data type should the radius be?
+1. What is the best way to get user input into a variable `radius` of that type?
+
+{{% notice orange "Warning" "rocket" %}} 
+ Be sure to create a main method to place your code within. It’s signature must be:
+
+ ```java
+ public static void main(String[] args)
+```
+{{% /notice %}}
+
+## B. The `Circle` Class
+
+Add a second Java file to your program to delegate the area calculation away from the printing task.
+
+1. Add a new class in your `studios.areaofacircle` package called `Circle`.
+1, Create a method called `getArea` inside of `Circle` that takes a `Double` `radius` as its only parameter and returns another `Double`, the result of the area calculation.
+   {{% notice green "Tip" "rocket" %}} 
+   ```java
+   public static Double getArea(Double radius) {
+   return 3.14 * radius * radius;
+   }
+   ```
+   {{% /notice %}}
+1. Back in `Area`, replace your area calculation line with a call to `Circle`.`getArea()`.
+
+{{% notice green "Tip" "rocket" %}} 
+ Check out the HelloMethods and Message example from [Static Methods](../reading/more-data-types/#static-methods) for a reference on how to use a class from another file.
+{{% /notice %}}
+
+Great work! Don't forget to save, `stage`, and `commit` your work up to GitHub.
```

Comment:
Suggest to tab over to match formatting from above.

Reply:
check!